plugin/functional-tests/src/test/groovy/com/novoda/gradle/release/ReleasePluginTest.groovy

```diff
@@ -103,11 +108,15 @@ class ReleasePluginTest {
 
     @Test
     void shouldGenerateJavadocs() {
+        skipTestWhen(configuration.android && configuration.gradleVersion >= GRADLE_4_10_1)
```

Comment:
Unfortunately I realised that the generation of the javadoc artifact is broken for recent versions of the Android Gradle Plugin, so I decided to skip these tests while we find a robust solution for it.